posix.mak

```diff
@@ -63,6 +63,8 @@ ROOT = $(ROOT_OF_THEM_ALL)/$(OS)/$(BUILD)/$(MODEL)
 DUB=dub
 GIT_HOME=https://github.com/dlang
 TOOLS_DIR=../tools
+DSCANNER_HASH=455cc3fe50e6d0742c866737b4ac24669d51a992
+DSCANNER_DIR=../dscanner-$(DSCANNER_HASH)
```

Comment:
> To avoid problems due to stale versions of DScanner, I'd suggest cloning it to a directory with the commit hash as part of the directory name, in the same way the dlang.org makefiles clone the stable versions of Phobos etc.

Excellent idea. Do you prefer long, short git hashes or simply don't care?